### DISCUSSION TITLE
Online Meeting Password: Unknown

### DISCUSSION THREAD

**Discussion by calfreycreative** [code4recovery/12-step-meeting-list#1490] [Q&A] [answered]
Hey, 

So when our PDF is generated, the password for the zoom meetings appears as unknown. I was wondering how this can be edited? I can't see a box to input the password seperately, so have been putting it in meeting notes but it doesn't seem to pull over to the WTF. 

Any help would be much appreciated, thanks!

**Comment by kiyote33** [comment, marked as answer]
Hi @calfreycreative,
Questions similar to this have been asked before. See topic #1372 for a more detailed response with an alternative suggestion, but the quick answer is that the PDF generator  doesn't support online meetings.

**Comment by calfreycreative** [comment]
Ah perfect thank you!

Kind Regards

Gemma Gracie
Creative Director
07568341900

On 20 Aug 2024 13:19, Wayne Parker ***@***.***> wrote:

Hi @calfreycreative<https://github.com/calfreycreative>,
Questions similar to this have been asked before. See topic #1372<https://github.com/code4recovery/12-step-meeting-list/discussions/1372> for a more detail response with an alternative suggestion, but the quick answer is that the PDF generator doesn't support online meetings.
